tools/local-network/README.md

```diff
@@ -3,7 +3,27 @@
 This directory contains scripts for running a local mobilecoin consensus network and a `mobilecoind` instance.
 The enclaves are built in pre-release mode, which provides debug symbols for debugging purposes, so it is only intended to be used for development and testing purposes.
 
-To run a local network, you will provide both the `IAS_API_KEY` and `IAS_SPID`, which you can obtain by registering with the [Intel SGX Portal](https://api.portal.trustedservices.intel.com/EPID-attestation
+To run a local network, you may provide both the `IAS_API_KEY` and `IAS_SPID`, which you can obtain by registering with the [Intel SGX Portal](https://api.portal.trustedservices.intel.com/EPID-attestation
+These are optional, and the script will provide defaults that work for local testing in software mode.
+
+## Quickstart
+
+The following sequence is a basic way to start a network locally.
+
+```
+$ export MC_LOG=info
+$ export LEDGER_BASE=$PWD/target/sample_data/ledger
+$ ./tools/local-network/bootstrap.sh
+$ ./tools/local-network/local_network.py --network-type dense5 &
+```
+
+Both `bootstrap.sh` and `local_network.py` will use cargo to build the required binaries.
+You only have to bootstrap once. To take down the network and start up a fresh one,
+you can just kill the `local_network.py` and restart it.
```

Comment:
I recently came across https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher (via Nick), which points out that words like 'simply' and 'just' are unhelpful at best
```suggestion
kill the `local_network.py` and restart it.
```